services/sleeper/requirements.in

```diff
@@ -0,0 +1,14 @@
+# tools
+pyyaml
+
+# tests
+coverage==4.5.1 # TODO: Downgraded because of a bug https://github.com/nedbat/coveragepy/issues/716
+docker
+jsonschema
+pytest~=5.3.5  # Bug in pytest-sugar https://github.com/Teemu/pytest-sugar/issues/187
```

Comment:
remove constrains on coverage and pytest . Not needed anymore in latest versions. 
Can you please also update it in the [cookie-cutter file](https://github.com/ITISFoundation/cookiecutter-osparc-service/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/requirements.in)